node/client/src/lib.rs

```diff
@@ -49,15 +50,19 @@ impl Client {
             index_col: storage::COL_BEACON_INDEX,
         };
         let runtime = Runtime::new(state_db.clone());
-        let genesis_root = runtime.apply_genesis_state(
-            &chain_spec.balances,
-            &chain_spec.genesis_wasm,
-        );
+        let genesis_root =
```

Comment:
nit: think we need to figure out how we feel about this formatting change

Reply:
I think the original one is better

Reply:
well, rust fmt...